pkg/importer/dataStream.go

```diff
@@ -899,10 +899,17 @@ func (d *DataStream) copy(dest string) error {
 	if util.GetAvailableSpace(d.ScratchDataDir) > int64(0) {
 		defer CleanDir(d.ScratchDataDir)
 	}
-	if d.isHTTPQcow2() {
-		err := d.convertQcow2ToRawStream(dest)
-		if err != nil {
-			return err
+	if d.isHTTPQcow2() || d.isIsoImage {
+		if d.isIsoImage {
+			err := StreamDataToFile(d.topReader(), dest)
+			if err != nil {
+				return err
+			}
+		} else {
+			err := d.convertQcow2ToRawStream(dest)
```

Comment:
Why can't we rename convertQcow2ToRawStream to something like convertFormatToRawStream(format, dest string)
And pass in the format we think it should be? Either qcow2 or raw if isIsoImage is true.